public/resources/stylesheets/page/_admin.scss

```diff
@@ -71,6 +79,7 @@
   font-weight: 600;
   background-image: linear-gradient(180deg, #33862f 0%, #33862f 100%);
```

Comment:
save the colors in variables, and check if that color are not already in the variables file